Sources/MapboxDirections/VisualInstructionComponent.swift

```diff
@@ -104,18 +109,48 @@ public extension VisualInstruction.Component {
             case svg
         }
         
-        /**
-         Initializes an image representation bearing the image at the given base URL.
-         */
-        public init(imageBaseURL: URL?) {
+        /// Initializes an image representation bearing the image at the given base URL.
+        /// - Parameters:
+        ///   - imageBaseURL: The image's base URL.
+        ///   - availableScales: The image’s available scale factors. If this argument is unspecified, the current screen’s native scale factor is used.
+        ///   - availableFormats: The available file formats of the image.
+        public init(imageBaseURL: URL?, availableScales: Set<CGFloat>? = nil, availableFormats: Set<Format>) {
             self.imageBaseURL = imageBaseURL
+            self.imageURL = nil
+            
+            let defaultScales: Set<CGFloat> = [ImageRepresentation.currentScale]
+            self.availableScales = availableScales ?? defaultScales
+            self.availableFormats = availableFormats
+        }
+        
+        /// Initializes an image representation bearing the image at the given URL.
+        /// - Parameters:
+        ///   - imageURL: The image URL.
+        ///   - availableScales: The image’s available scale factors. If this argument is unspecified, the current screen’s native scale factor is used.
```

Comment:
> If this argument is unspecified, the current screen’s native scale factor is used.

This is true of `imageURL(scale:format:)`, but I think it would be better for a `nil` `availableScales` to mean “don’t append a scale to the URL”.